marshaling_test.go

```diff
@@ -256,27 +256,125 @@ func (s *Zuite) TestStructScan_skipFieldsWithNoTag() {
 	require.Equal(s.T(), "ignore me", data.Ignore)
 }
 
-func (s *Zuite) TestStructScan_slicesNotSupported() {
+func (s *Zuite) TestStructScan_slices() {
+	var data struct {
+		Texts []string `ws:"slice_t"`
```

Comment:
To me, testing matrix is

dest: `[]string` and `[]*string`, possibly `*[]string` and `*[]*string`

values: empty, filled, with undefined

and we'd want to test the cross of dest x values